src/App.tsx

```diff
@@ -1,14 +1,65 @@
 /* eslint-disable max-len */
-import React from 'react';
+import React, { useEffect, useState } from 'react';
 import 'bulma/css/bulma.css';
 import '@fortawesome/fontawesome-free/css/all.css';
 
 import { TodoList } from './components/TodoList';
-import { TodoFilter } from './components/TodoFilter';
+import { Status, TodoFilter } from './components/TodoFilter';
 import { TodoModal } from './components/TodoModal';
 import { Loader } from './components/Loader';
+import { Todo } from './types/Todo';
+import { getTodos } from './api';
+
+interface Filters {
+  query: string;
+  status: Status;
+}
+
+const getFilteredTodos = (todos: Todo[], { query, status }: Filters) => {
+  let filteredTodos = [...todos];
+  const normalizeQuery = query.trim().toLowerCase();
```

Comment:
i would not trim querty
Imagine - the user looks for "[space]and" and finds "stand"